app/lanes.go

```diff
@@ -0,0 +1,80 @@
+package app
+
+import (
+	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client"
+
+	signerextraction "github.com/skip-mev/block-sdk/adapters/signer_extraction_adapter"
+	"github.com/skip-mev/block-sdk/block/base"
+	defaultlane "github.com/skip-mev/block-sdk/lanes/base"
+	mevlane "github.com/skip-mev/block-sdk/lanes/mev"
+)
+
+const (
+	maxTxPerMEVLane     = 500  // this is the maximum # of bids that will be held in the app-side in-memory mempool
+	maxTxPerDefaultLane = 3000 // all other txs
+)
+
+var (
+	defaultLaneBlockspacePercentage = math.LegacyMustNewDecFromStr("0.90")
+	mevLaneBlockspacePercentage     = math.LegacyMustNewDecFromStr("0.10")
```

Comment:
mev lane is 10% of block space :eyes:

Reply:
Can we rename this to top of block auction lane?